City-of-Helsinki/drupal-hdbt
# UHF-X: Javascript error fixes

How to test:

1. Install latest sote-instance to your local. It would be recommended to create it from a database dump so that you have navigation available.
2. Go to a landing page that has subitems on the navigation. If you are using a dump you should go to https://helfi-sote.docker.so/fi/sosiaali-ja-terveyspalvelut/terveydenhoito/terveysasemat
3. This page used to give errors on the console that say something in these lines: `Uncaught TypeError: Cannot read properties of undefined (reading 'getElementsByClassName') at HTMLDocument.<anonymous> (sidebar-menu-toggle.min.js?v=1.x:1)`
4. `composer require drupal/hdbt:dev-UHF-X_javascript_error_fixes`
5. `make drush-cr`
6. Reload the same page and the error should be gone.
7. Now go to some page that has sidebar navigation and navigation should have multiple items and two levels. Test that the sidebar toggle still works so that you can show and hide the items from the navigation.
